be/src/exec/csv_scanner.cpp

```diff
@@ -296,14 +296,18 @@ Status CSVScanner::_parse_csv_v2(Chunk* chunk) {
             if (_counter->num_rows_filtered++ < 50) {
                 std::stringstream error_msg;
                 error_msg << "Value count does not match column count. "
-                          << "Expect " << _num_fields_in_csv << ", but got " << row.columns.size();
+                          << "Expect " << _num_fields_in_csv << ", but got " << row.columns.size() << "."
+                          << "Column delimiter " << _parse_options.column_delimiter << ","
+                          << "Row delimiter " << _parse_options.row_delimiter << ".";
```

Comment:
Does it need escaping?
otherwise, there will be a new line in the log if row_delimiter is \n. similar for column_delimiter.